README.md

```diff
@@ -154,28 +204,29 @@ $bodyContent = $emogrifier->emogrifyBodyContent();
 
 #### Options
```

Comment:
Now that this is referring to `CssInliner`, I think it can go back to being a level 3 heading, and perhaps moved before the “Using the legacy Emogrifier class” section.

Reply:
Done.